src/dashboard/src/components/auth/AuthState.tsx

```diff
@@ -79,13 +77,6 @@ export const AuthState: React.FC<IAuthState> = ({ showName }) => {
             } else {
               setUserinfo(userinfo);
             }
-            setAdminUsers(
```

Comment:
Removed as it was causing issues.  Not certain why.  But we don't need it now that we don't care about optimistic concurrency when editing users.